coremltools/models/utils.py

```diff
@@ -1004,6 +1004,60 @@ def _convert_to_float(feature):
             convert_double_to_float_multiarray_type(model_spec)
 
 
+def compile_model(model: _Union['_ct.models.MLModel', str, _Model_pb2.Model]):
+    """
+    Compiles a Core ML model.
+
+    Parameters
+    ----------
+
+    model: str, Model_pb2 or MLModel
+
+        str - path to model to compile
+
+        Model_pb2 - spect to model to compile
+
+        MLModel - instantiated Core ML model to compile
+
+    Returns
+    -------
+
+    str : path to compiled model directory
+
+    See Also
+    --------
+
+    ``coremltools.models.CompiledMLModel``
+    """
+    # Check environment
+    if _macos_version() < (10, 13):
```

Comment:
why this check?
Is there any related change in CoreML, besides the python binding?

Reply:
There is no related change in the Core ML Framework.

macOS 10.13 was the first version to have Core ML. This code will not work on a macOS older than 10.13, or on Linux/Windows.

Reply:
Oh!
Okay!
I misread it as `13.0` lol